cli/tests/cli-reference@.md.snap

```diff
@@ -802,6 +802,8 @@ When none of the `--destination`, `--insert-after`, or `--insert-before` argumen
 
 When any of the `--destination`, `--insert-after`, or `--insert-before` arguments are provided, the roots of the specified commits will be duplicated onto the destination indicated by the arguments. Other specified commits will be duplicated onto these newly duplicated commits. If the `--insert-after` or `--insert-before` arguments are provided, the new children indicated by the arguments will be rebased onto the heads of the specified commits.
 
+The descriptions of the duplicated commits can be customized with the `templates.duplicate_description`.
```

Comment:
Maybe
```suggestion
By default, the duplicated commits retain the description of the originals. This can be customized with the `duplicate_description` template.
```

Reply:
Looks good, but I wouldn't remove `templates.` prefix from the config name.

Reply:
https://jj-vcs.github.io/jj/latest/config/#default-description does, which is why I did.

Could do
```suggestion
By default, the duplicated commits retain the description of the originals. This can be customized with `templates.duplicate_description`.
```
but I would like them to be consistent.

Reply:
In the command help context, it's not obvious where to put the `duplicate_description` config value. That's why I prefer fully-qualified name here.